.vscode/settings.json

```diff
@@ -1,19 +1,19 @@
 {
   "workbench.colorCustomizations": {
-    "activityBar.activeBackground": "#c749cd",
-    "activityBar.activeBorder": "#c8c137",
-    "activityBar.background": "#c749cd",
-    "activityBar.foreground": "#e7e7e7",
-    "activityBar.inactiveForeground": "#e7e7e799",
-    "activityBarBadge.background": "#c8c137",
+    "activityBar.activeBackground": "#8330FF",
+    "activityBar.activeBorder": "#46eca1",
+    "activityBar.background": "#100C53",
+    "activityBar.foreground": "#F9FCFD",
+    "activityBar.inactiveForeground": "#8330FF",
+    "activityBarBadge.background": "#46ECA1",
     "activityBarBadge.foreground": "#15202b",
-    "statusBar.background": "#ac31b2",
-    "statusBar.foreground": "#e7e7e7",
-    "statusBarItem.hoverBackground": "#c749cd",
-    "titleBar.activeBackground": "#ac31b2",
-    "titleBar.activeForeground": "#e7e7e7",
-    "titleBar.inactiveBackground": "#ac31b299",
-    "titleBar.inactiveForeground": "#e7e7e799"
+    "statusBar.background": "#100C53",
```

Comment:
It seems like VS Code sometimes resets the theme on its own. Not sure why. You might want to revert that.

Reply:
Yes, my impression is that it's related to the Peacock extension.

Anyway, this change here is actually on purpose. The theme was accidentally reset in some previous PR, here I'm changing it back to our Stadtpuls colors.